module/signer.go

```diff
@@ -24,5 +24,8 @@ type AggregatingSigner interface {
 type ThresholdSigner interface {
 	ThresholdVerifier
 	Sign(msg []byte) (crypto.Signature, error)
-	Combine(size uint, shares []crypto.Signature, indices []uint) (crypto.Signature, error)
+}
+
+type SignerStore interface {
```

Comment:
It may be clearer to include `beacon` or `threshold` in the name, to avoid the confusion with the staking signers.